src/app/screens/Dashboard/screens/UseState/components/SquareFunctional/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+
+import { SquareCoords } from '~utils/types';
+
+import { printCoords, initial } from '../../utils';
+
+function SquareFunctional() {
```

Comment:
```suggestion
function Square() {
```
We could keep de `ClassComponent` suffix, but I'd name functional ones as usual.